packages/next-swc/crates/next-api/src/project.rs

```diff
@@ -504,15 +505,16 @@ impl Project {
                 node_root.join("chunks".to_string()),
                 node_root.join("assets".to_string()),
                 node_build_environment(),
+                next_mode.runtime_type(),
```

Comment:
The execution context is for postcss/webpack execution and I guess it should always use the dev runtime

Reply:
Let's revisit that in the follow-up PR that introduces the production runtime as for now they both use the dev runtime anyway.

Curious to learn more about why this should use the dev runtime though

Reply:
Opened a PR for this: https://github.com/vercel/turbo/pull/7721/files